adserver/templates/adserver/publisher/payouts.html

```diff
@@ -0,0 +1,52 @@
+{% extends "adserver/base.html" %}
+{% load humanize %}
+{% load i18n %}
+
+
+{% block title %}{% trans 'Publisher payouts' %} - {{ publisher }}{% endblock %}
+
+
+{% block breadcrumbs %}
+  {{ block.super }}
+  <li class="breadcrumb-item"><a href="{% url 'publisher_main' publisher.slug %}">{{ publisher }}</a></li>
+  <li class="breadcrumb-item active">{% trans 'Payouts' %}</li>
+{% endblock breadcrumbs %}
+
+
+{% block content_container %}
+
+<h1>{% block heading %}{% trans "Publisher payouts" %}{% endblock heading %}</h1>
+
```

Comment:
We should probably have some copy here explaining our payout policies, and links to some docs perhaps? Definitely need at least some introductory copy.

Reply:
Where should those docs live? Not in the ad server, I'd think.